### ISSUE TITLE
Compatibility with pymbar 4

### ISSUE THREAD

**Issue by mattwthompson** [choderalab/openmmtools#629]
Are there plans to update this to work with the new pymbar? Is this something that I could help out with?

**Comment by ijpulidos** [comment]
I don't think we have immediate plans for this, but I believe this is something we do want to support. Any help is appreciated! Thanks!
